### PR TITLE
Make both redspot & rust-analyzer happy

### LINKED ISSUE #1: Change project structure
With current project, i.e.  

```sh
tree uniswap-dex-example
```

    .
    ├── Cargo.toml
    ├── README.md
    ├── tests
    └── uniswap-v2
        ├── Cargo.toml
        ├── lib.rs
        ├── contracts
        ├── impls
        └── traits

LSP (e.g. rust-analyzer) do not anything when it comes to editing the files under `contracts` with  

```sh
cat uniswap-dex-example/Cargo.toml
```

    [workspace]
    members = [
        "uniswap-v2/contracts/*",
    ]
    
    exclude = [
        "uniswap-v2"
    ]

I suggest to take contracts directory out of the project folder, `uniswap-v2`.  
E.g.  

```sh
tree uniswap-dex-example
```

    .
    ├── Cargo.toml
    ├── README.md
    ├── contracts
    ├── tests
    └── uniswap-v2
        ├── Cargo.toml
        ├── lib.rs
        ├── impls
        └── traits

with  

```sh
cat uniswap-dex-example/Cargo.toml
```

    [workspace]
    members = [
        "contracts/*",
    ]

In this way, LSP does duly expected job.

### PR DESCRIPTION
**Pull request by HyunggyuJang**
Closes #1 
## Details
- Lock toolchain in redspot (recent toolchain raises [an error upon contract compilation](https://substrate.stackexchange.com/questions/4785/errore0158-when-testing-default-contract-from-flipper)
- Remove not (yet) used project from root's workspace, where only contracts should reside
- Move logics of uniswap-v2 project to its own directory to make redspot & rust-analyzer (LSP) work properly at the same time
## Propose
- [ ] Setup typescript config file, tsconfig.json
- [ ] Cleanup typescript files to pacify error messages from ts-server

### CODE REVIEW

File: redspot.config.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export default {
     defaultNetwork: 'development',
     contract: {
         ink: {
-            toolchain: 'nightly',
+            toolchain: 'nightly-2022-07-24',
```
Comment (PierreOssun) [inline redspot.config.ts:10]: If you lock toolchain here can you also add an rust-toolchain.toml ?
https://github.com/AstarNetwork/Astar/blob/master/rust-toolchain.toml

**Review by PierreOssun — COMMENTED** [review]
Thank you for the PR @HyunggyuJang

File: redspot.config.ts (hunk 1 above)
Comment (HyunggyuJang) [reply redspot.config.ts:10]: Fixed as of https://github.com/AstarNetwork/astar-wasm-contracts-examples/pull/7/commits/cec6767869c39e6d6664f176b9a6bc5295d6af9a

**Review by PierreOssun — APPROVED** [review]
LGTM

**Pull request merged**